### ISSUE TITLE
Log header info showing SPEK1024 while set on CRSF

### ISSUE THREAD

**Issue by mmakam2** [betaflight/blackbox-log-viewer#298]
I'm using CRSF crossfire, and the log header is showing SPEK1024..

![image](https://user-images.githubusercontent.com/25559970/55827303-7f2e9480-5acf-11e9-919b-24cec8046011.png)

**Comment by McGiverGim** [comment]
Yes, it seems the protocol list is outdated. I will fix it tomorrow. Thanks for reporting!
